.github/workflows/ci.yml

```diff
@@ -10,7 +10,13 @@ on:
 
 jobs:
   static-analysis:
-    runs-on: ubuntu-20.04
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      fail-fast: false
+      matrix:
+        os: ["ubuntu-20.04", "macos-10.15", "macos-11"]
```

Comment:
this needs a collective job in the end called "Static Analysis" else the required workflow constraint is not met.

Reply:
Ah I thought it was because the name didn't seem to match 100% (it is "CI Pipeline / Static Analysis" vs "Static Analysis"). Will check how to do this properly then